actions/errata.py

```diff
@@ -110,7 +115,7 @@ def update(errataidlist, cache_only=None):
         erratas = s.errata.getErrataNamesById(system_id, errataidlist)
         errata_names = [tup[1] for tup in erratas]
 
-        return packages.patch_install(errata_names, cache_only)
+        return packages.patch_install(errata_names, False, cache_only, allow_vendor_change)
```

Comment:
The updatestack_only has nothing to do with this feature. Why should we add it when we say always "False"